modules/metastorage/build.gradle

```diff
@@ -38,6 +39,7 @@ dependencies {
     implementation project(':ignite-failure-handler')
     implementation project(':ignite-metrics')
     implementation project(':ignite-system-disaster-recovery-api')
+    implementation project(':ignite-catalog')
```

Comment:
And this one?

Reply:
It doesn't look correct to me: it's catalog to depend on metastorage (metastorage-api actually) and not vice-versa.

Reply:
Leftovers from deleted code, removed.